integration_tests/test_get_requests.py

```diff
@@ -51,3 +52,12 @@ def test_queries(function_type: str, session):
 
     r = get(f"/{function_type}/queries")
     assert r.json() == {}
+
+
+@pytest.mark.benchmark
+def test_trailing_slash(session):
+    r = requests.get("http://localhost:8080/trailing")
```

Comment:
Hey @VishnuSanal , instead of `requests.get`, you can use the `get` method we have imported from `from integration_tests.helpers.http_methods_helpers import get`

Reply:
@sansyrox That strips the trailing slash. Tests always pass!

Reply:
Ah, okay. That makes sense. Could you please comment at line 59? And then I will do a final review before merge 😄